west.yml

```diff
@@ -219,7 +219,7 @@ manifest:
       groups:
         - hal
     - name: hal_silabs
-      revision: d191d981c4eb20c0c7445a4061fcdbcfa686113a
+      revision: pull/46/head
```

Comment:
Hi! Please change the revision to `8bc20509e3b1dcff9a70e153110697130eb83549`. Thanks!

Reply:
Hi, @fkokosinski, I have updated the revision, please review and merge the PR, Thanks.